mobile/src/main/java/ch/epfl/sdp/mobile/ui/i18n/English.kt

```diff
@@ -101,6 +102,10 @@ object English : LocalizedStrings {
 
   override val playOnlineGames = "Online games"
 
+  override val puzzlePlayingAs = { color: String -> "Playing as $color" }
+  override val puzzleUnsolvedPuzzles = "Unsolved puzzles"
+  override val puzzleListContentDescription = "puzzle_list"
```

Comment:
Content descriptions may be read by screen readers, so they must be like normal text

```suggestion
  override val puzzleListContentDescription = "Puzzles list"
```

Reply:
https://github.com/epfl-SDP/android/pull/300/commits/347d4e97d50faf2807747bded27a3dd06cde7d85